packages/grafana-ui/src/components/TimePicker/_TimePicker.scss

```diff
@@ -7,6 +7,17 @@
   }
 }
 
+.active-time-picker {
```

Comment:
we try to follow BEM (we are not super great at it though). So if this is overriding another class it should be a modifier class instead.  time-picker-button-select--active 

then you can define it in the same class definition using `&--active {`

Reply:
As of my other comment in this PR, I suggest migrating TimePicker to Emotion